test/largescale/tomo/ray_transform_slow_test.py

```diff
@@ -29,87 +29,127 @@
 # Internal
 import odl
 import odl.tomo as tomo
+from odl.tomo.util.testutils import (skip_if_no_astra, skip_if_no_astra_cuda,
+                                     skip_if_no_scikit)
 
 
 pytestmark = odl.util.skip_if_no_largescale
 
 
 # Find the valid projectors
-projectors = ['helical cuda']
-if tomo.ASTRA_AVAILABLE:
-    projectors += ['par2d cpu', 'cone2d cpu']
-if tomo.ASTRA_CUDA_AVAILABLE:
-    projectors += ['par2d cuda', 'cone2d cuda',
-                   'par3d cuda', 'cone3d cuda']
+projectors = [skip_if_no_astra('par2d astra_cpu uniform'),
```

Comment:
This syntax is lightyears better than the previous syntax since it gives actual skipped messages if the tests are not run.